applications/nrf5340_audio/src/audio/sw_codec_select.c

```diff
@@ -236,12 +236,19 @@ int sw_codec_decode(uint8_t const *const encoded_data, size_t encoded_size, bool
 
 		switch (m_config.decoder.channel_mode) {
 		case SW_CODEC_MONO: {
-			ret = sw_codec_lc3_dec_run(encoded_data, encoded_size,
-						   LC3_PCM_NUM_BYTES_MONO, 0, pcm_data_mono,
-						   (uint16_t *)&pcm_size_session, bad_frame);
-			if (ret) {
-				return ret;
+
```

Comment:
Nitpick: We don't have newlines after the other instances of 'case' in this switch

Reply:
Fair enough, I'll remove it. :)

Reply:
You're right, fixed.